### PR TITLE
fix(badge): show badge when mentioned even if channel is muted

### LINKED ISSUE #8723: Muted communities don't produce a badge when mentioned
# Bug Report

## Description

Muted communities should show a badge if you get mentioned

## Steps to reproduce

1. Mute channel A in an community
2. Change section to the personal chat
3. Have another account mention you in the community in channel A
4. No indication whatsoever.

### PR DESCRIPTION
**Pull request by jrainville**
Fixes #8723

### CODE REVIEW

File: src/app/modules/main/chat_section/module.nim [hunk 1]
```diff
@@ -719,18 +719,18 @@ method onNewMessagesReceived*(self: Module, sectionIdMsgBelongsTo: string, chatI
...
-    # No need to update the badge nor send a notification
-    return
-
   # Badge notification
   let messageBelongsToActiveSection = sectionIdMsgBelongsTo == self.controller.getMySectionId() and 
     self.controller.getMySectionId() == self.delegate.getActiveSectionId()
   let messageBelongsToActiveChat = self.controller.getActiveChatId() == chatIdMsgBelongsTo
   if(not messageBelongsToActiveSection or not messageBelongsToActiveChat):
-    let hasUnreadMessages = unviewedMessagesCount > 0
+    let hasUnreadMessages = (not chatDetails.muted and unviewedMessagesCount > 0) or unviewedMentionsCount > 0
```
Comment (caybro) [inline src/app/modules/main/chat_section/module.nim:727]: Well isn't that the same as before? :)

File: src/app/modules/main/chat_section/module.nim [hunk 2]
```diff
@@ -719,18 +719,18 @@ method onNewMessagesReceived*(self: Module, sectionIdMsgBelongsTo: string, chatI
...
     self.updateBadgeNotifications(chatIdMsgBelongsTo, hasUnreadMessages, unviewedMentionsCount)
 
+  if (chatDetails.muted):
+    # No need to update the badge nor send a notification
```
Comment (caybro) [inline src/app/modules/main/chat_section/module.nim:731]: You already updated the badge

File: src/app/modules/main/chat_section/module.nim (hunk 1 above)
Comment (jrainville) [reply src/app/modules/main/chat_section/module.nim:727]: Look again, there is `unviewedMessagesCount` and `unviewedMentionsCount` (messages vs mentions) :wink:

**Review by osmaczko — APPROVED** [review]

File: src/app/modules/main/chat_section/module.nim (hunk 2 above)
Comment (jrainville) [reply src/app/modules/main/chat_section/module.nim:731]: Opps yeah I'll update the comment

**Comment by jrainville** [comment]
I messed up my rebase, do not merge

**Pull request merged**